labels.json

```
@@ -87,6 +87,14 @@
     "name": "🧹 status: ticket work required",
     "color": "666666"
   },
+  {
+    "name": "🙅 status: discontinued",
+    "color": "666666"
```

Why use the same color here and in line 96? I would suggest using this color `#0033CC`.
```suggestion
    "color": "#0033CC"
```

---

Cool, i followed the previous repo-labels and used the same colors! This blue looks better!